docs/sample_inventories/migration_sample_scram.yml

```diff
@@ -0,0 +1,52 @@
+---
+##
+## The following is an example inventory file for Migration of ZK cluster to Kraft cluster
+## steps to run:
+## 1. Run command "ansible-playbook -i <inv.yml> confluent.platform.all" to setup a ZK cluster
+## 2. Once ZK cluster setup is complete, uncomment "kraft_migration: true" and kafka_controller section.
+## 3. If your sasl_protocol is scram or scram256, uncomment the kafka_controller_sasl_protocol variable and set it to "plain". You can set it to 'kerberos' as well if needed after setting up kerberos server accordingly.
+## 4. Run command "ansible-playbook -i <inv.yml> confluent.platform.ZKtoKraftMigration"
+## 5. Once Kraft cluster is up and running, Remove "kraft_migration: true" and Zookeeper section from the inventory file.
+
+all:
+  vars:
+    ansible_connection: ssh
+    ansible_user: ec2-user
+    ansible_become: true
+    ansible_ssh_private_key_file: /home/ec2-user/guest.pem
+    ansible_python_interpreter: /usr/bin/python3
+    #    kafka_controller_sasl_protocol: plain
```

Comment:
uncommented part should first setup the scram + zk cluster. 
Then steps should take them to scram+ kraft cluster